amp.php

```diff
@@ -14,102 +14,111 @@
  */
 
 /**
- * Print admin notice regarding having an old version of PHP.
+ * Errors encountered while loading the plugin.
  *
- * @since 0.7
+ * This has to be a global for the same of PHP 5.2.
+ *
+ * @var \WP_Error $_amp_load_errors
  */
-function _amp_print_php_version_admin_notice() {
-	?>
-	<div class="notice notice-error">
-		<p>
-			<?php
-			printf(
-				/* translators: %s: required PHP version */
-				esc_html__( 'The AMP plugin requires PHP %s. Please contact your host to update your PHP version.', 'amp' ),
-				'5.4+'
-			);
-			?>
-		</p>
-	</div>
-	<?php
-}
+global $_amp_load_errors;
+
+$_amp_load_errors = new \WP_Error();
+
 if ( version_compare( phpversion(), '5.4', '<' ) ) {
-	add_action( 'admin_notices', '_amp_print_php_version_admin_notice' );
-	return;
+	$_amp_load_errors->add(
+		'insufficient_php_version',
+		sprintf(
+			/* translators: %s: required PHP version */
+			__( 'The AMP plugin requires PHP %s; please contact your host to update your PHP version.', 'amp' ),
```

Comment:
Just noticed this. We should turn this semicolon into a dot again in order to not make existing translations fuzzy.

Reply:
See https://github.com/ampproject/amp-wp/pull/2208